website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -6,115 +6,63 @@ weight: 3
 
 This guide walks through an example of building a simple memcached-operator powered by Ansible using tools and libraries provided by the Operator SDK.
 
-## Create a new project
+## Prerequisites
 
-After [installing the Operator SDK CLI][install-guide] and
-[ansible operator prerequisites][ansible-install-guide], use the CLI to create a
-new Ansible-based memcached-operator project:
+- [docker][docker-tool] version 17.03+.
+- [kubectl][kubectl-tool] version v1.11.3+.
+- [Ansible Operator SDK Installation][ansible-operator-install] v1.0.0+
+- Access to a Kubernetes v1.11.3+ cluster.
 
-```sh
-$ mkdir memcached-operator && cd memcached-operator
-$ operator-sdk init --plugins=ansible --domain=example.com
-$ operator sdk create api --group=cache --version=v1alpha1 --kind=Memcached --generate-role
-```
-
-This creates the memcached-operator project specifically for watching the
-Memcached resource with APIVersion `cache.example.com/v1alpha1` and Kind
-`Memcached`.
-
-To learn more about the project directory structure, see [project
-layout][layout-doc] doc.
-
-#### Operator scope
+## Creating an Operator
 
-Read the [operator scope][operator-scope] documentation on how to run your operator as namespace-scoped vs cluster-scoped.
+In this section we will:
+  - extend the Kubernetes API with a [Custom Resource Definition][custom-resources] that allows users to create `Memcached` resources.
+  - create a manager that updates the state of the cluster to the desired state defined by `Memcached` resources.
```

Comment:
I do not think that it is required at all. If we are using operators we are extending k8s api always. 
I would prefer the docs less verbose as possible and more like how to's
At least if we need to describe something is a specific feature or just what brings value for the reader.